modules/nw-control-dns-records-public-hosted-zone-aws.adoc

```diff
@@ -6,7 +6,7 @@
 [id="nw-control-dns-records-public-hosted-zone-aws_{context}"]
 = Creating DNS records on an public hosted zone for AWS by using Red Hat External DNS Operator
 
-You can create DNS records on a public hosted zone for AWS by using the Red Hat External DNS Operator.
+You can create DNS records on a public hosted zone for AWS by using the Red Hat External DNS Operator. You can use the same instructions to create DNS records on a public hosted zone for AWS GovCloud.
```

Comment:
I believe AWS GovCloud's Route53 still doesn't support public hosted zones. So, better to not mention `public` for GovCloud.

Reply:
Addressed- Thanks for the review!